unity-renderer/Assets/UIComponents/Scripts/Components/GridContainer/GridContainerComponentView.cs

```diff
@@ -65,7 +65,7 @@ public interface IGridContainerComponentView
     /// Adds a new item in the grid.
     /// </summary>
     /// <param name="item">An UI component.</param>
-    void AddItem(BaseComponentView item);
+    void AddItemWithResize(BaseComponentView item);
```

Comment:
Please, lets update it in the documentation we have about `GridContainer` UI Component in notion: https://www.notion.so/decentraland/UI-Components-Reference-f467a8d05d594718a2c12809dcfaf132?p=5a29ed52bfc34987b627e1fd7ec546b5&pm=s